### PR TITLE
Migrate to eslint

### PR DESCRIPTION
**Pull request by julien-deramond**
Followed https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint

```
$ node_modules/.bin/ng add @angular-eslint/schematics
ℹ Using package manager: npm
✔ Found compatible package version: @angular-eslint/schematics@12.6.1.
✔ Package information loaded.

The package @angular-eslint/schematics@12.6.1 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.

    All @angular-eslint dependencies have been successfully installed 🎉

    Please see https://github.com/angular-eslint/angular-eslint for how to add ESLint configuration to your project.

UPDATE package.json (3895 bytes)
✔ Packages installed successfully.
```

```
$ node_modules/.bin/ng g @angular-eslint/schematics:convert-tslint-to-eslint Orange-Boosted-Angular
? Would you like to remove TSLint and its related config if there are no TSLint projects remaining after this conversion? Yes
? Would you like to ignore the existing TSLint config? Recommended if the TSLint config has not been altered much as it makes the new ESLint config cleaner. Yes

    INFO: We are now installing the "tslint-to-eslint-config" package into a tmp directory to aid with the conversion

    This may take a minute or two...

CREATE .eslintrc.json (1015 bytes)
UPDATE angular.json (5649 bytes)
```

Did the same thing for the 2 other sub-projects:
* `$ node_modules/.bin/ng g @angular-eslint/schematics:convert-tslint-to-eslint Orange-Boosted-Angular-e2e`
* `$ node_modules/.bin/ng g @angular-eslint/schematics:convert-tslint-to-eslint ng-boosted`

### CODE REVIEW

**Comment by julien-deramond** [comment]
Remaining error to handle before the review:
```
/Users/xx/Orange-Boosted-Angular/src/app/demos/toasts-container.component.ts
  20:3  error  Use @HostBinding or @HostListener rather than the `host` metadata property (https://angular.io/styleguide#style-06-03)  @angular-eslint/no-host-metadata-property

✖ 1 problem (1 error, 0 warnings)

Lint errors found in the listed files.
```

**Comment by julien-deramond** [comment]
Please tell me if I need to reintegrate some specific tslint rules. I made the choice to restart from scratch here with the recommended rules for an Angular project.

File: package.json [hunk 1]
```diff
@@ -28,6 +28,11 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^12.1.1",
+    "@angular-eslint/builder": "12.6.1",
```
Comment (Lausselloic) [inline package.json:31]: maybe use `^12.6.1` to allow minor update like for other angular dependencies?

File: package.json [hunk 2]
```diff
@@ -45,12 +50,14 @@
     "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.8",
     "@types/node": "^14.6.4",
+    "@typescript-eslint/eslint-plugin": "4.28.2",
```
Comment (Lausselloic) [inline package.json:53]: same here? propose to allow 4.*

**Review by Lausselloic — COMMENTED** [review]
small improvement to do on package.json to allow minor version, all the other points sounds good for me

**Comment by Lausselloic** [comment]
> 
> 
> Please tell me if I need to reintegrate some specific tslint rules. I made the choice to restart from scratch here with the recommended rules for an Angular project.

It's okay for me. Let's start with fresh and up to date rules!

**Review by Lausselloic — APPROVED** [review]

**Comment by julien-deramond** [comment]
@BehnooshShiva Sounds OK to you?

**Pull request merged**